index.html

```diff
@@ -69,14 +69,23 @@
     const input = document.getElementById("input");
     const output = document.getElementById("output");
     const pianomap = document.getElementById("pianomap");
+    const outType = document.getElementById("outputType");
     form.addEventListener("submit", function(e) {
         const map = maps[pianomap.value];
+        const oT = outType.value
+        var photoCode = convert(map, input.value.split('')).join('');
         try {
-            output.value = convert(map, input.value.split('')).join('');
+            output.value = photoCode;
         } catch(e) {
             output.value = `Error while converting! ${e}\n\nDoes your song span too many octaves?`;
+            oT += "errorLOL"
         }
         e.preventDefault();
+        if (oT == "photo") {
+            photoCode = '[["PolaroidPhoto",[],{"Phrase":"' + convert(map, input.value.split('')).join('') + '"}]]';
+            photoCode = btoa(photoCode)
+            output.value = photoCode;
+        }
```

Comment:
Everything else seems fine but this looks very sloppy. Could you rework this to be less destructive to the original logic of the applet? If not I would be willing to do it.

Reply:
i'll try to clean it up

Reply:
fixed it a bit, it looks a little better now. please point out what I could try to fix some more.